src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -23,16 +23,17 @@
  */
 
 /*
- *
- *  (C) Copyright IBM Corp. 1999 All Rights Reserved.
- *  Copyright 1997 The Open Group Research Institute.  All rights reserved.
```

Comment:
OpenJDK copyrights should not be removed.

Reply:
So, should I just revert it to what it was?